src/components/Users/Users.js

```diff
@@ -1,69 +1,71 @@
 import React from 'react';
 
-// import * as axios from 'axios';
+import * as axios from 'axios';
 
-// axios.get('https://social-network.samuraijs.com/api/1.0/users').then(response => {
-//     props.setUsers()
-// });
+import userPhoto from '../../assets/images/unnamed.jpg'
 
 import styles from './Users.module.css';
 
-const Users = (props) => {
-  const span1 = (user) => {
-    const { photoUrl, id, followed } = user;
+class Users extends React.Component {
+    constructor(props) {
+        super(props);
+        axios.get('https://social-network.samuraijs.com/api/1.0/users')
```

Comment:
Why are you making a request in the constructor and not in ComponentDIdMount.

Reply:
/fxd